terraform/aws/components/gp2gp/variables.tf

```diff
@@ -153,6 +153,17 @@ variable "gp2gp_service_target_request_count" {
   default = 1200
 }
 
+variable "gp2gp_extract_cache_bucket" {
```

Comment:
what is this variable for? Are we going to use other bucket that the one we create?

Reply:
earlier I thought to use this variable in tfvars file to specify bucket name, but later on i dropped idea but forgot to take it off. Will take it off now!

Reply:
entry removed.